packages/contracts/src/networks.ts

```diff
@@ -60,11 +70,18 @@ export const SPHINX_NETWORKS: Array<SupportedNetwork> = [
     chainId: BigInt(1),
     rpcUrl: () =>
       `https://eth-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`,
-    etherscan: {
-      apiURL: 'https://api.etherscan.io/api',
-      browserURL: 'https://etherscan.io',
-      envKey: 'ETH_ETHERSCAN_API_KEY',
-      blockExplorer: 'Etherscan',
+    blockexplorers: {
+      etherscan: {
+        apiURL: 'https://api.etherscan.io/api',
+        browserURL: 'https://etherscan.io',
+        envKey: 'ETH_ETHERSCAN_API_KEY',
+      },
+      blockscout: {
+        apiURL: 'https://eth.blockscout.com/api',
+        browserURL: 'https://eth.blockscout.com/',
+        envKey: 'ETH_BLOCKSCOUT_API_KEY',
+        selfHosted: false,
+      },
```

Comment:
I'm still testing verification on the newly supported blockscout networks, so these fields may change. I don't think that needs to block the review though.